### ISSUE TITLE
HTML text is missing

### ISSUE THREAD

**Issue by barabadzhi** [sharazam/printpdf#198]
The WASM demo linked in the repo as well as native lib loosing all text content from HTML.
The actual text seems OK, but `shaped_words` / `glyphs` come out empty.

**Comment by fschutt** [comment]
Yeah, this is an issue I'm debugging atm, it seems that `allsorts::subset::subset(font, glyphs)` reorders the glyph IDs so `lookup_glyph(char as u32)` fails. This should be fixed upstream however.

**Comment by fschutt** [comment]
The issue was that the subsetted font didn‘t contain some glyphs because I didn’t encode the WINDOWS_1252 codepage properly. Should be fixed tomorrow.

**Comment by fschutt** [comment]
The subsetting actually works now, but the HTML rendering is pretty broken, both on WASM and on native.

I had other things to do so this will have to wait for 0.9.

**Comment by fschutt** [comment]
This is now properly done on master, issue can be closed. HTML layout works as far as I tested it.
